apps/astarte_appengine_api/lib/astarte_appengine_api/device/queries.ex

```diff
@@ -285,19 +284,66 @@ defmodule Astarte.AppEngine.API.Device.Queries do
         endpoint_id,
         path,
         value_timestamp,
-        reception_timestamp
+        reception_timestamp,
+        opts
       ) do
-    property_table = String.replace(interface_descriptor.storage, "datastreams", "properties")
+    insert_path(
+      db_client,
+      device_id,
+      interface_descriptor,
+      endpoint_id,
+      path,
+      value_timestamp,
+      reception_timestamp,
+      opts
+    )
+  end
 
+  def insert_path_into_db(
+        db_client,
+        device_id,
+        %InterfaceDescriptor{storage_type: :one_object_datastream_dbtable} = interface_descriptor,
+        endpoint_id,
+        path,
+        value_timestamp,
+        reception_timestamp,
+        opts
+      ) do
+    insert_path(
+      db_client,
+      device_id,
+      interface_descriptor,
+      endpoint_id,
+      path,
+      value_timestamp,
+      reception_timestamp,
+      opts
+    )
+  end
+
+  defp insert_path(
+         db_client,
+         device_id,
+         interface_descriptor,
+         endpoint_id,
+         path,
+         value_timestamp,
+         reception_timestamp,
+         _opts
+       ) do
     # TODO: use received value_timestamp when needed
     # TODO: :reception_timestamp_submillis is just a place holder right now
+    insert_statement = """
+    INSERT INTO individual_properties
```

Comment:
use
```elixir
property_table =
  if interface_descriptor.aggregation == :individual do
    String.replace(interface_descriptor.storage, "datastreams", "properties")
  else
    "individual_properties"
  end
```
instead of hardcoding `individual_proerties`